src/Whoops/Run.php

```diff
@@ -230,7 +230,7 @@ public function writeToOutput($send = null)
      * @param  Exception $exception
      * @return string    Output generated by handlers
      */
-    public function handleException(Exception $exception)
+    public function handleException($exception)
```

Comment:
Yes... I did this hastily to get Whoops working in PHP 7 as I test other code bases, but I suppose that's a poor excuse.